packages/cli/src/commands/init/errors/DirectoryAlreadyExistsError.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import path from 'path';
+
+export default class DirectoryAlreadyExistsError extends Error {
+  constructor(directory: string) {
+    super(
+      `Cannot initialize new project because directory ".${
+        path.sep
```

Comment:
It looks weird to have this as a local reference including `./`. I would just remove that tbh.